app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Admin Dashboard</h1>
+<p> <%= link_to "Admin Invoices", admin_invoices_path %><p>
+<p> <%= link_to "Admin Merchants", admin_merchants_path %><p>
+
+<ul id="dashboard-customers">
+  <p>Top 5 Customers: </p>
+    <% @customers.top_five_customers.each do |customer| %>
+  <li>
+   Name: <%= customer.first_name %> <%= customer.last_name %> | Successful Transactions: <%= customer.transaction_ct("success") %>
+  </li>
+<% end %>
+</ul>
+
+<ul id="dashboard-incomplete_invoices">
+  <p>Incomplete Invoices: </p>
+    <% @invoices.incomplete_invoices.reverse_order.each do |invoice| %>
```

Comment:
It appears the `incomplete_invoices` method was already ordering the results by ascending order (this is the default). `reverse_order` changes the order to descending when we want to keep the order ascending per the User Story: "And I see that the list is ordered from oldest to newest".

See comment on line 164 of the `features/admin/index_spec.rb`

Reply:
Thank you! I got those mixed up so will fix locally and push up the changes.